doc/threat_model/threat_model.md

```diff
@@ -192,10 +192,15 @@ locked by blowing the fuses.
 
 #### Mitigations
 
+- USB port attacks - boot protocol:
+
+  - Firmware now exits to CPU trap state instead of exiting to an
+    eternal loop on errors.
+
 - Software attacks: 
 
-  - Access to RAM outside of physical RAM stops the CPU instead of
-    wrapping.
+  - Access to RAM outside of physical RAM forces the CPU into a trap
+    state that requires a reboot instead of wrapping.
```

Comment:
I would like to rephrase to something like: "Instead or wrapping, access to RAM outside (eller 'beyond') the address range of the physical RAM...

Basically, I want to move "instead of wrappinng" to beginning. It looks weird with "requires a boot instead of wrapping".